test-case/check-volume-levels.sh

```diff
@@ -35,6 +35,7 @@ OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg file, default value is env TPLG: $'
 OPT_HAS_ARG['t']=1         OPT_VAL['t']="$TPLG"
 
 func_opt_parse_option "$@"
+setup_kernel_check_point
```

Comment:
This code looks like it was missing but the commit message does not really make sense to me.
One optional but useful thing for the commit message would be: how you found that this was missing.

Reply:
I updated again, can you review?